src/format/opamFile.mli

```diff
@@ -1026,18 +1034,32 @@ module Repos_config: sig
 end
 
 module Switch_config: sig
-  type t = {
+  type t = private {
```

Comment:
This is not correlated to this PR, it should go on its own PR.

Reply:
Commit removed (and conflict resolved)